src/lazy/binary/raw/v1_1/immutable_buffer.rs

```diff
@@ -361,28 +361,6 @@ impl<'a> ImmutableBuffer<'a> {
     ) -> IonResult<LazyRawBinaryValue_1_1<'a>> {
         todo!();
     }
-
-    /// Runs the provided parsing function on this DataSource's buffer.
-    /// If it succeeds, marks the `DataSource` as ready to advance by the 'n' bytes
-    /// that were consumed.
-    /// If it does not succeed, the `DataSource` remains unchanged.
-    pub(crate) fn try_parse_next<F: Fn(Self) -> IonResult<Option<LazyRawBinaryValue_1_1<'a>>>>(
```

Comment:
This method was actually broken with the removal of `bytes_to_skip`, but luckily nothing else used it. I noticed as I was working through adding similar `bytes_to_skip` functionality for sequences.